packages/flutter_tools/test/general.shard/ios/ios_project_migration_test.dart

```diff
@@ -339,6 +340,94 @@ keep this 2
       });
     });
 
+    group('Xcode default build location', () {
+      MemoryFileSystem memoryFileSystem;
+      BufferLogger testLogger;
+      MockIosProject mockIosProject;
+      File xcodeProjectWorkspaceData;
+
+      setUp(() {
+        memoryFileSystem = MemoryFileSystem();
+        xcodeProjectWorkspaceData = memoryFileSystem.file('contents.xcworkspacedata');
+
+        testLogger = BufferLogger(
+          terminal: AnsiTerminal(
+            stdio: null,
+            platform: const LocalPlatform(),
```

Comment:
nit. can you use a `FakePlatform` instead?

Reply:
I didn't know about `BufferLogger.test()` when I first wrote this file.  Good eye.

Reply:
I spent a whole day once removing `const LocalPlatform()`'s from tests, so it instantly jumps out at me.